packages/install-expo-modules/src/plugins/ios/withSwiftVersion.ts

```diff
@@ -0,0 +1,26 @@
+import { type ConfigPlugin, withXcodeProject } from '@expo/config-plugins';
+import type { XcodeProject } from 'xcode';
+
+export function setSwiftVersionIfNotPresent(
+  swiftVersionToSet: string,
```

Comment:
I wonder if I should be providing the `swiftVersionToSet` or detecting it from other targets?

Reply:
right now it's going to be set to 5.0